docs/src/content/docs/resources/showcase.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Starlight Showcase
+description: Discover sites built with Starlight and community tools that extend Starlight!
+sidebar:
+  label: Site Showcase
+---
+
+:::tip[Add your own!]
+Have you built a Starlight site, plugin, or a tool for Starlight?
```

Comment:
Should the references to plugins and tools be removed from the frontmatter description and the first aside considering this page now only lists websites built with Starlight? And I guess the question would be if a similar aside should be added to the `resources/plugins.mdx` page?

Reply:
Great observation! I think it would be nice to have one aside for every page 🚀. One for sites, content and plugins/integrations 💜